### PR TITLE
Brick!

### PR DESCRIPTION
**Pull request by geralddejong**
We have bricks.

### CODE REVIEW

File: src/build/brick_proto.rs [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        lines.extend(self.faces
+            .into_iter()
+            .map(|(joints, face_name, spin)|
+                format!("        ({joints:?}, {face_name:?}, {spin:?}),")));
+        lines.push("    ],".to_string());
+
+        lines.push("}".to_string());
+        lines.join("\n")
+    }
+}
```
Comment (mitchelljustin) [inline src/build/brick_proto.rs:44]: Fold this into the `impl Brick` below

Comment (geralddejong) [reply src/build/brick_proto.rs:44]: Yes

File: src/build/tenscript/build_phase.rs [hunk 2]
```diff
@@ -198,6 +198,21 @@ impl BuildPhase {
         !self.buds.is_empty()
     }
 
+    pub fn init(&mut self, fabric: &mut Fabric) {
+        let BuildPhase { seed, root, .. } = &self;
+        match root {
+            None => {
+                self.twist(fabric, seed.needs_double(), seed.spin(), None);
```
Comment (mitchelljustin) [inline src/build/tenscript/build_phase.rs:205]:
```suggestion
                // TODO: this should be called attach_brick() and it should take a BrickName 
                self.twist(fabric, seed.needs_double(), seed.spin(), None);
```

File: src/build/tenscript/bootstrap.scm [hunk 3]
```diff
@@ -27,6 +27,7 @@
           (face D- (grow 11 (scale .92) (mark :halo-end))))))
     (shape
       (join :halo-end)
+      (remove-shapers) ; should automatically happen before vulcanize
```
Comment (mitchelljustin) [inline src/build/tenscript/bootstrap.scm:30]:
```suggestion
      (remove-shapers) ; TODO: should automatically happen before vulcanize
```

File: src/build/tenscript/plan_runner.rs [hunk 4]
```diff
@@ -43,12 +43,12 @@ impl PlanRunner {
         let (next_stage, countdown) = match self.stage {
             Initialize => {
                 self.build_phase.init(fabric);
-                (GrowApproach, 1500)
+                (GrowApproach, 200)
```
Comment (mitchelljustin) [inline src/build/tenscript/plan_runner.rs:46]: Rapido!

Comment (geralddejong) [reply src/build/tenscript/plan_runner.rs:46]: Mucho!

File: src/build/tenscript/shape_phase.rs [hunk 5]
```diff
@@ -136,7 +136,7 @@ impl ShapePhase {
                 let joints = self.marked_middle_joints(fabric, &faces);
                 match (joints.as_slice(), faces.as_slice()) {
                     (&[alpha_index, omega_index], &[alpha_face, omega_face]) => {
-                        let interval = fabric.create_interval(alpha_index, omega_index, Link::pull(0.3));
+                        let interval = fabric.create_interval(alpha_index, omega_index, Link::pull(0.01));
```
Comment (mitchelljustin) [inline src/build/tenscript/shape_phase.rs:139]: Why?

Comment (geralddejong) [reply src/build/tenscript/shape_phase.rs:139]: Kind of arbitrary nonzero

**Review by mitchelljustin — APPROVED** [review]

**Pull request merged**